docs/data/data-grid/demo/PopularFeaturesDemo.js

```diff
@@ -271,19 +306,46 @@ function CustomToolbar() {
 function RowDemo(props) {
   const { row } = props;
   const theme = useTheme();
-  const gridBgColor = theme.palette.mode === 'dark' ? '#000' : '#fff';
-  const panelColor = theme.palette.mode === 'dark' ? 'transparent' : '#efefef';
 
   return (
-    <Box sx={{ py: 2, backgroundColor: panelColor }}>
+    <Box
+      sx={{
+        py: 6,
+        bgcolor: theme.palette.mode === 'dark' ? '#141A1F' : 'grey.50', // dark color is the branding theme's primaryDark.800
```

Comment:
We can't use `primaryDark.800` here because the demo is sandboxed and has no access to the branding theme, right?

Reply:
Yup, precisely! That's why I ended up importing Material Design colors, too.